docs/source/overview.rst

```diff
@@ -71,12 +71,9 @@ Under consideration: Fab might get a config file format in the future.
 Multiprocessing
 ---------------
 
-Steps have access to multiprocessing methods. The :class:`~fab.steps.mp_exe.MpExeStep` class captures common aspects
-of any step that passes a list of artefacts through a command-line tool.
-(The existence of this class is currently under debate.)
-The Step class includes a multiprocessing utility method called :meth:`~fab.steps.Step.run_mp` which steps can call
-from their :meth:`~fab.steps.Step.run` method to pass a collection of artefacts through a processor function
-in parallel.
+Steps have access to multiprocessing methods.
+The Step class includes a multiprocessing helper method called :meth:`~fab.steps.Step.run_mp` which steps can call
```

Comment:
You might want to reconsider the name of this helper method, because I guarentee a few people will be unhappy with it.

Reply:
Oh, thanks @GiorgiaLine. It's not clear which kind of names we need to avoid here or what the objection would be in this case. Are you able to elaborate please? Does it help that _:meth:`~fab.steps.Step.run_mp`_ is rendered by Sphinx as just a clickable `run_mp`, and that a step writer would access the function as `self.run_mp`?

Reply:
Never mind, coming back to this with fresh eyes I realise I've misread it and the variable name itself isn't meth.